### PR TITLE
Created job-detail component and managed styling of all pages in global css file(#23m9z3e)

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

**Comment by azkyakhan** [comment]
job-detail

**Comment by azkyakhan** [comment]
localhost_8102_orders

**Comment by azkyakhan** [comment]
https://user-images.githubusercontent.com/82817616/158598624-a940f4b7-d6b8-46f6-9ccb-451bbfbdfa74.mp4

File: src/components/JobDetail.vue [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      personCircleOutline
+    };
+  }
+});
+</script>
+
+<style scoped>
+section {
+  overflow: hidden;
+  width: 547px;
```
Comment (dt2patel) [inline src/components/JobDetail.vue:98]: lets also add a min width here of 355px
```suggestion
  min-width: 355px;
```

Comment (dt2patel) [inline src/components/JobDetail.vue:98]: We can change default width to flex 1 so it grows with size of container now.

**Review by dt2patel — CHANGES_REQUESTED** [review]

**Pull request closed without merging**